c2_utils/src/mfx_va_allocator.cpp

```diff
@@ -616,4 +623,25 @@ mfxStatus MfxVaFrameAllocator::MapGrallocBufferToSurface(buffer_handle_t gralloc
     return mfx_res;
 }
 
+mfxStatus MfxVaFrameAllocator::TouchSurface(VASurfaceID surface)
```

Comment:
Can you help to explain why this is needed? look like it only write the 1st buffer to 0 and do nothing else. Will it bring any performance improvement?

Reply:
Hi @dyang23 , 

With this workaround, the first frame can come out or display earlier for videos with large resolution. It can reduce latency for the first video frame. Otherwise, we will wait more time to see the first frame come out. It is ported from omx https://cactus-source.iind.intel.com/source/xref/CELADON-MASTER-R2/vendor/intel/mediasdk_omx_il/omx_utils/src/mfx_omx_vaapi_allocator.cpp#567.

Reply:
Ok, so most likely it's issue of MSDK or low-level driver, is the latest onevpl still have this issue and need WA?

Reply:
Yes, I think so. The oneVPL we are using is not the latest release from https://github.com/oneapi-src/oneVPL-intel-gpu/. With our current oneVPL, we still need this WA for the latency issue. Will check this issue again after we upgrade oneVPL to the latest release. TBD.

Reply:
Ok, let's keep it, Thanks,